### DISCUSSION TITLE
Having problem with workbench tree indent

### DISCUSSION THREAD

**Discussion by meichengg** [microsoft/vscode-discussions#690] [Extension Development QnA]
Hi guys, i'm having an issue with `"workbench.tree.indent"`, eventhough i set this key in JSON settings (the key will be underlined somehow) or change the number in normal Settings (when i hit Save, it reverts to the default value), it won't affect.

![image](https://github.com/microsoft/vscode-discussions/assets/25204246/c9438aff-88c6-4443-ace4-881e83bbbaa2)

**Comment by isidorn** [comment]
This discussion repo is a place for VS Code extension authors. I am closing this as out of scope, and I suggest to ask this question on Stack Overflow.
